modules/core/src/main/java/com/illposed/osc/transport/udp/OSCPortIn.java

```diff
@@ -64,6 +66,141 @@
 	private final OSCParserFactory parserFactory;
 	private final List<OSCPacketListener> packetListeners;
 
+	private static OSCPacketDispatcher getDispatcher(
+		final List<OSCPacketListener> listeners
+	)
+	{
+		for (OSCPacketListener listener : listeners) {
+			if (listener instanceof OSCPacketDispatcher) {
+				return (OSCPacketDispatcher)listener;
+			}
+		}
+
+		return null;
+	}
+
+	private static OSCPacketListener defaultPacketListener() {
+		OSCPacketDispatcher dispatcher = new OSCPacketDispatcher();
+		// HACK: We do this, even though it is against the OSC (1.0) specification,
+		// because this is how it worked in this library until Feb. 2015., and thus
+		// users of this library expect this behaviour by default.
+		dispatcher.setAlwaysDispatchingImmediately(true);
+
+		return dispatcher;
+	}
+
+	private static List<OSCPacketListener> defaultPacketListeners() {
+		List<OSCPacketListener> listeners = new ArrayList<OSCPacketListener>();
+		listeners.add(defaultPacketListener());
+		return listeners;
+	}
+
+	public static class Builder {
+		private OSCParserFactory parserFactory;
+		private List<OSCPacketListener> packetListeners;
+		private SocketAddress local;
+		private SocketAddress remote;
+
+		private void addDefaultPacketListener() {
+			if (packetListeners == null) {
+				packetListeners = new ArrayList<OSCPacketListener>();
+			}
+
+			packetListeners.add(defaultPacketListener());
+		}
+
+		public OSCPortIn build() throws IOException {
+			if (local == null) {
+				throw new IllegalArgumentException(
+					"Missing local socket address / port."
+				);
```

Comment:
formatting (here and a few lines below); the same like previously with method heads